contributing/code/tests.rst

```diff
@@ -16,14 +16,13 @@ In any case, it's a good practice to run tests locally before submitting a
 Before Running the Tests
 ------------------------
 
-To run the Symfony test suite, `install PHPUnit`_ 4.2 (or later) first. Then,
-install the external dependencies used during the tests, such as Doctrine, Twig
-and Monolog. To do so, :doc:`install Composer </cookbook/composer>` and execute
-the following:
+To run the Symfony test suite, install the external dependencies used during the
+tests, such as Doctrine, Twig and Monolog. To do so,
+:doc:`install Composer </cookbook/composer>` and execute the following:
 
 .. code-block:: bash
 
-    $ composer install
+    $ composer update
```

Comment:
When one works on a patch and a composer file is changed, only update works, that's why it's more generic